### PR TITLE
Add README

### PR DESCRIPTION
**Pull request by 0x49**
Just a simple README file for repo description

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Junjin's Blog
+===
```
Comment (junjinp) [inline README.md:2]: Some comments
```suggestion
var a = b;
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+Junjin's Blog
```
Comment (junjinp) [inline README.md:1]: A single comment

**Review by junjinp — CHANGES_REQUESTED** [review]
LGTM

File: README.md (hunk 1 above)
Comment (0x49) [reply README.md:2]: I don't think we should do that. Using "===" to signify that "Junjin's Blog" is a header

File: README.md [hunk 3]
```diff
@@ -0,0 +1,4 @@
...
+===
+
+This is the repo for [Junjin's Blog](https://junjinpu.com)
```
Comment (junjinp) [inline README.md:4]: LG!!!

**Review by junjinp — APPROVED** [review]

**Pull request merged**